config.yaml

```diff
@@ -13,6 +13,15 @@ repos:
       rhel8_package: rust-afterburn
       rhel9_package: rust-afterburn
 
+  bootupd:
+    url: https://github.com/coreos/bootupd
+    vars:
+      git_repo: bootupd
+      crate: bootupd
```

Comment:
I haven't been updating this because there's no reason to.

Reply:
The idea is to unify our procedures across our repos to make sure that once we do an emergency update, anybody on the team can do it safely following the instructions. This also reduces the chance of errors during releases.

Reply:
Yeah fair enough...it just feels silly to publish things to crates.io which one shouldn't actually use as a Rust dependency or `cargo install`.

Reply:
I'm fine not doing that if that's not needed and we can tweak the release checklist. I appears that we're doing that right now from https://github.com/coreos/bootupd/blob/main/.github/ISSUE_TEMPLATE/release-checklist.md